### ISSUE TITLE
Can't set an NFS Share to read only

### ISSUE THREAD

**Issue by steven-ellis** [arensb/ansible-truenas#14]
Looks like an error in plugins/modules/sharing_nfs.py

Existing code 
```
            if readonly is not None and export_info['readonly'] != readonly:
                arg['ro'] = readonly
```

Corrected code
```
            if readonly is not None and export_info['ro'] != readonly:
                arg['ro'] = readonly
```

**Comment by arensb** [comment]
Do you have these two blocks of code reversed?

Also, since it looks as though you have a fix, would you mind putting in a pull request?

**Comment by steven-ellis** [comment]
Done. And fixed the comment above

**Comment by arensb** [comment]
Fixed in [v1.9.3](https://github.com/arensb/ansible-truenas/releases/tag/v1.9.3).
